joomla/joomla-cms
# [Sql azure] Normalize joomla install sample testing sql file

### Summary of Changes

Normalizes all sql azure joomla **sample_testing.sql** install file.
- always use `SET IDENTITY_INSERT [#__xxxx] ON/OFF;`
- `INSERT` to `INSERT INTO`
- Quote all table names and db field names
- Always use `'` to quote string values.
- Code style
### Testing Instructions

Mainly code review. (to many changes need to check code in raw) 
https://github.com/andrepereiradasilva/joomla-cms/blob/patch-26/installation/sql/sqlazure/sample_testing.sql

If anyone could test an install of **sample testing** 3.6.x would be great. Don't have sql azure to test,
### Documentation Changes Required

None.

@waader @alikon please test.
